tutorials/getting-started-with-flow/04__creating-a-form-using-plain-java/content.adoc

```diff
@@ -139,22 +140,22 @@ public CustomerForm(MainView mainView) {
 ...
 
 private void save() {
-    Customer customer = binder.getBean(); <1>
-    service.save(customer); <2>
-    mainView.updateList(); <3>
-    setCustomer(null); <4>
+    Customer customer = binder.getBean(); 
+    service.save(customer); 
+    mainView.updateList(); 
+    setCustomer(null); 
 }
 ----
-<1> Gets the `customer` instance that was bound to the input fields of the form.
+* `getBean` gets the `customer` instance that was bound to the input fields of the form.
 
-<2> Performs the save action in the backend.
+* `service.save(customer)` performs the save action in the backend.
 
-<3> Updates the list of customers in the main view.
+* `updateList` updates the list of customers in the main view.
 
-<4> Hides this form.
-
-Similarly, you can implement the delete action as follows:
+* `setCustomer(null)` hides the form data.
```

Comment:
@ruthstocks This and any other occurrence of "form data" should be reverted to "form" as well.

Reply:
Sorry, fixing now

Reply:
Thanks!